radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/network/SimulationNetwork.java

```diff
@@ -171,6 +148,10 @@ private SimulatedNetworkImpl(BFTNode node) {
 				.filter(msg -> msg.receiver.equals(node))
 				.groupBy(MessageInTransit::getSender)
 				.flatMap(groupedObservable ->
+					channelCommunication
+						.transform(groupedObservable.getKey(), node, groupedObservable)
+						.map(MessageInTransit::getContent)
+					/*
```

Comment:
This commented out code needs some explanation of why it is commented out, and what needs to be done to bring it back.